continuum/tasks/utils.py

```diff
@@ -24,3 +25,33 @@ def split_train_val(dataset: TaskSet, val_split: float = 0.1) -> Tuple[TaskSet,
     val_dataset = TaskSet(x_val, y_val, t_val, dataset.trsf, dataset.data_type)
 
     return train_dataset, val_dataset
+
+
```

Comment:
je vois plutot concatenate comme une methode de taskset que comme une fonction exterieur.
Mais c'est juste un point de vue

Reply:
Both are valid and useful I guess, one is in-place while the other is side-effect free.

Reply:
ok